app/components/file-uploader.js

```diff
@@ -96,7 +96,9 @@ export default Ember.Component.extend({
         },
         complete(_, dropzone, file) {
             // Complete is called when swapping out files for some reason...
-            if (file.xhr === undefined) return;
+            if (file.xhr === undefined) {
```

Comment:
@abought, please do not reformat code unless it does not meet the style guide. A single line if blocks is acceptable per the AirBnB style guide we are following.

https://github.com/airbnb/javascript/blob/master/README.md#blocks--braces